.buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py

```diff
@@ -110,7 +110,8 @@ def build_repo_wide_pyright_steps() -> List[CommandStep]:
             "pip install -e python_modules/dagster[pyright] -e python_modules/dagster-pipes",
             "make pyright",
         )
-        .on_test_image(AvailablePythonVersion.get_default())
+        # 3.11 (current default) can't support the entire huge pyright environment as of 2024-01-29
```

Comment:
Do we have a list of what we're waiting on for support?

Reply:
this turned out to be a red herring, 3.11 works